zxlive/proof_actions.py

```diff
@@ -197,6 +206,19 @@ def custom_rule(graph: Graph, vertices: List[VT], lhs_graph: nx.Graph, rhs_graph
 
     return etab, vertices_to_remove, [], True
 
+def get_vertex_positions(graph, rhs_graph, boundary_vertex_map):
+    pos_dict = {v: (graph.qubit(m), graph.row(m)) for v, m in boundary_vertex_map.values()}
```

Comment:
So one thing I want to point out, is that .qubit() is a y coordinate and .row() is an x coordinate. So this mapping you have is (y,x) which might lead to confusion later. It's fine here though as you using it consistently.
Otherwise this looks like a very neat solution.

Reply:
Good point, let me change that.

Reply:
Fixed in https://github.com/Quantomatic/zxlive/pull/74/commits/495749992f5e5fbc37431990c51d6ffccbc83c3d